lighthouse-core/test/audits/dobetterweb/uses-http2-test.js

```diff
@@ -61,4 +61,22 @@ describe('Resources are fetched over http/2', () => {
       assert.ok(auditResult.displayValue === '');
     });
   });
+
+  it('results are correct when some requests are handled by service worker', () => {
+    const clonedNetworkRecords = JSON.parse(JSON.stringify(networkRecords));
+    clonedNetworkRecords.forEach(record => {
+      // convert http 1.1 to service worker requests
+      if (record.protocol === 'http/1.1') {
+        record.fetchedViaServiceWorker = true;
+      }
+    });
+
+    return UsesHTTP2Audit.audit(
+      getArtifacts(clonedNetworkRecords, URL), {computedCache: new Map()}
+    ).then(auditResult => {
+      assert.equal(auditResult.rawValue, false);
+      assert.ok(auditResult.displayValue.match('1 request not'));
+      assert.equal(auditResult.details.items[0].url, 'https://webtide.com/wp-content/themes/clean-retina-pro/library/js/tinynav.js?ver=4.5.4');
```

Comment:
why doesn't this one go away too? is it not `http/1.1`?

Reply:
friendly ping :)

Reply:
will have a look today :)

Reply:
```js
{
  protocol: 'http/1.0',
  url: 'https://webtide.com/wp-content/themes/clean-retina-pro/library/js/tinynav.js?ver=4.5.4'
}
```
Protocol is http/1.0 not http/1.1 so should be flagged.

Reply:
oh gotcha, can we leave a comment so it's explained or just convert the test to flip all to sw or something? :)

Reply:
i'll add a comment, not adding all requests to sw was added because of this comment:
https://github.com/GoogleChrome/lighthouse/pull/7197#discussion_r255352918

Reply:
Oh my b, yeah comment is ace 👌